java/ql/lib/ext/experimental/s3-transfer-manager.model.yml

```diff
@@ -0,0 +1,13 @@
+extensions:
+  - addsTo:
+      pack: codeql/java-all
+      extensible: experimentalSinkModel
+    data:
+      - ["software.amazon.awssdk.transfer.s3.model","ResumableFileUpload",true,"serializeToFile","(Path)","","Argument[0]","path-injection","manual"]
```

Comment:
@am0o0 I think the experimentalSinkModel has a different column count. But let's wait for the CodeQL team, maybe these additions shouldn't be placed there.

Reply:
Hey @am0o0 👋 

The last column in the `experimentalSinkModel` would be used to filter them to only be used for a particular experimental query. But that's not something that can be done here (as the PR does not come with a new experimental query).

So I think it makes the most sense if you just replace `experimentalSinkModel` with `sinkModel` in the YAML files.

/cc @owen-mc Thanks!